internal/pkg/manifest/validate_test.go

```diff
@@ -3242,6 +3246,18 @@ func TestValidateWindows(t *testing.T) {
 			in:          validateWindowsOpts{},
 			wantedError: nil,
 		},
+		"error if readonlyfs is true": {
+			in: validateWindowsOpts{
+				readOnlyFS: aws.Bool(true),
+			},
+			wantedError: errors.New(`'ReadOnlyFS' can not be set to true when deploying a Windows container`),
+		},
+		"should return nil readonlyfs is false or nil": {
```

Comment:
```suggestion
		"should return nil if readonlyfs is false or nil": {
```

Reply:
The test checks for `aws.Bool(false)` but not when the value is `nil`. Maybe we can rename it `should return nil if readonly_fs is false`. 

We can rename the above test case from `should return nil efs not specified` to `should return nil when no fields are specified`